iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings

```diff
@@ -1525,7 +1527,7 @@
 
 "type.boundary.administrative.11" = "Frontera administrativa";
 
-"type.boundary.administrative.2" = "Frontera de país";
+"type.boundary.administrative.2" = "National Border";
```

Comment:
That should not happen. Did you update our twine submodule?

Reply:
Fixed